mne/inverse_sparse/mxne_optim.py

```diff
@@ -110,7 +110,7 @@ def prox_l1(Y, alpha, n_orient):
     [ True  True False False]
     """
     n_positions = Y.shape[0] // n_orient
-    norms = np.sqrt(np.sum((np.abs(Y) ** 2).T.reshape(-1, n_orient), axis=1))
+    norms = np.sqrt((Y * Y.conj()).real.T.reshape(-1, n_orient).sum(axis=1))
```

Comment:
why did you have the `np.abs(Y)` at all, then? BTW the `.conj()` adds *very* little overhead when the data are real...